<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autoFolder</class>
 <widget class="QDialog" name="autoFolder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>autoFolder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btnTargetFolder">
     <property name="text">
      <string>1. Куда поместить проект</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnTemplate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>2. Файл шаблона</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>3. Имя проекта:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtProjectName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
